pkg/data/management/role_data.go

```diff
@@ -199,10 +199,12 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 		addRule().apiGroups("management.cattle.io").resources("clusterroletemplatebindings").verbs("get", "list", "watch")
 
 	rb.addRoleTemplate("Manage Cluster Catalogs", "clustercatalogs-manage", "cluster", false, false, true).
-		addRule().apiGroups("management.cattle.io").resources("clustercatalogs").verbs("*")
+		addRule().apiGroups("management.cattle.io").resources("clustercatalogs").verbs("*").
+		addRule().apiGroups("catalog.cattle.io").resources("clusterrepos").verbs("*")
 
 	rb.addRoleTemplate("View Cluster Catalogs", "clustercatalogs-view", "cluster", false, false, false).
-		addRule().apiGroups("management.cattle.io").resources("clustercatalogs").verbs("get", "list", "watch")
+		addRule().apiGroups("management.cattle.io").resources("clustercatalogs").verbs("get", "list", "watch").
```

Comment:
https://github.com/rancher/rancher/blob/ec31e6456d4068899460bb83640b63c83eaa72c6/pkg/data/management/role_data.go#L225

https://github.com/rancher/rancher/blob/ec31e6456d4068899460bb83640b63c83eaa72c6/pkg/data/management/role_data.go#L221
looks like it matches!